package.json

```diff
@@ -5,11 +5,18 @@
   "version": "1.6.0",
   "license": "MIT",
   "dependencies": {
+    "@gatsbyjs/reach-router": "^2.0.1",
```

Comment:
Why did you add so many new dependencies?

Reply:
Initially while setting up the project , I had run into these module not found errors and so I had installed them. I kind of realised that it would be an issue, let me try and setup things again.

Reply:
I've used npm instead of pnpm , so there's no module not found errors now. They had arised due to webpack being unable to resolve the dependencies. There is a [plugin](https://www.gatsbyjs.com/plugins/gatsby-plugin-pnpm/) to solve this but I used npm rather than going with the plugin.